### PR TITLE
[OSDEV-1346] - Refactor index mapping and sorting for name and address of moderation event

### PR DESCRIPTION
**Pull request by VadimKovalenkoSNF**
Follow-up PR for [OSDEV-1346](https://opensupplyhub.atlassian.net/browse/OSDEV-1346)
Refactor index mapping and sorting for `cleaned_data.name` and `cleaned_data.address` of moderation event to prevent error:

```
'search_phase_execution_exception', 'Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead.
```

[OSDEV-1346]: https://opensupplyhub.atlassian.net/browse/OSDEV-1346?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by roman-stolar — APPROVED** [review]
LGTM

File: doc/release/RELEASE-NOTES.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Code/API changes
 * [OSDEV-1453](https://opensupplyhub.atlassian.net/browse/OSDEV-1453) - The `detail` keyword instead of `message` has been applied in error response objects for V1 endpoints.
-* [OSDEV-1346](https://opensupplyhub.atlassian.net/browse/OSDEV-1346) - Disabled null values from the response of the OpenSearch. Disabled possible null `os_id`, `claim_id` and `source` from `PATCH api/v1/moderation-events/{moderation_id}` response.
+* [OSDEV-1346](https://opensupplyhub.atlassian.net/browse/OSDEV-1346) - Disabled null values from the response of the OpenSearch. Disabled possible null `os_id`, `claim_id` and `source` from `PATCH api/v1/moderation-events/{moderation_id}` response. Updated index mapping for `cleaned_data.name` and `cleaned_data.address` for `GET api/v1/moderation-events`.
```
Comment (vladsha-dev) [inline doc/release/RELEASE-NOTES.md:22]: Could you please state the reason for the update? What value did your changes bring?
Please reflect this in the PR description as well.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: doc/release/RELEASE-NOTES.md (hunk 1 above)
Comment (VadimKovalenkoSNF) [reply doc/release/RELEASE-NOTES.md]: Updated release notes.

**Review by mazursasha1990 — APPROVED** [review]
LGTM

**Comment by VadimKovalenkoSNF** [comment]
Abandon this PR in favor of PR: [Fix OpenSearch index mapping updates](https://github.com/opensupplyhub/open-supply-hub/pull/445 )

**Pull request closed without merging**